java/google/registry/keyring/api/DummyKeyringModule.java

```diff
@@ -42,6 +42,30 @@
  * with all attempted outgoing connections failing because the supplied dummy credentials aren't
  * valid. For a real system that needs to connect with external services, you should replace this
  * module with one that loads real credentials from secure sources.
+ * 
+ * <p>The dummy PGP keyrings are created using gnupg1/pgp1 roughly like the following (using
+ * gnupg2/pgp2 is an exercise left for the developer):
+ * <code>
+ * {@code
+ * # mkdir gpg
+ * # chmod 700 gpg
+ * # gpg1 --homedir gpg --gen-key <<<EOF
```

Comment:
What is `gpg1`?  Why isn't this just a normal `gpg` command?

Reply:
normal gpg is gpg2 (though on older distributions like Ubuntu 14.04 it is still gpg1). Bouncy castle should be working with keys generated in either version. Specifying a version to use here is helpful because likely they have different flags.

Reply:
Who knew there were two GnuPG versions, right? Perhaps this is part of why PGP still doesn't catch-on. [This StackExchange question](https://superuser.com/a/655250) explains more about the versions.

I'm no GnuPG expert so there may be a better way. I mainly wanted to capture the steps I used for creating the keys since I'd done it once before and forgotten how. I used `gpg1` because I didn't want to pollute my keyring, which is what `--homedir` should do, but it was failing due to the "agent" not running. I failed to work around that and discovered that GnuPG 1 didn't have that problem.